_component_design/breadcrumbs.md

```diff
@@ -18,30 +18,59 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/breadcrumbs/
 
 <section class="static-section" markdown="1">
 
-Use breadcrumbs on Rackspace sub-pages to let the user know where they are in the page hierarchy. Do not use breadcrumbs on the first layer of any multi-layered module, only use breadcrumbs on the second level of nesting and deeper within a module.
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use breadcrumbs when you want to improve web application wayfinding and navigation on sites that organize content hierarchically. For example, breadcrumbs help users better identify where they are. This is particularly true of sites that use top navigation.
```

Comment:
nix "For example, breadcrumbs help users better identify where they are"

Reply:
Done.